test/server/config_validation/config_fuzz_test.cc

```diff
@@ -0,0 +1,46 @@
+#include <fstream>
+
+#include "envoy/config/bootstrap/v2/bootstrap.pb.h"
+#include "envoy/config/bootstrap/v2/bootstrap.pb.validate.h"
+
+#include "common/network/address_impl.h"
+#include "common/thread_local/thread_local_impl.h"
+
+#include "server/config_validation/server.h"
+#include "server/drain_manager_impl.h"
+#include "server/hot_restart_nop_impl.h"
```

Comment:
I think a bunch of these headers aren't needed either; they are vestigial from parts that were removed in `server_fuzz_test`.